packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -248,10 +370,6 @@ export const SelectTree = function SelectTree( {
 								comboBoxProps={ {
 									className:
 										'woocommerce-experimental-select-control__combo-box-wrapper',
-									role: 'combobox',
```

Comment:
These props didn't belong here, but on the inputProps, that's why they are being removed